tests/test_custom_models.py

```diff
@@ -3178,12 +3178,19 @@ def test_mixed_adapter_batches_lora_mlp(self, mlp_lora):
 
     def test_mixed_adapter_batches_lora_different_target_layers(self, mlp_lora):
         base_model = MLP().to(self.torch_device).eval()
-        # target different lora layers
         config0 = LoraConfig(target_modules=["lin0"], init_lora_weights=False)
         config1 = LoraConfig(target_modules=["lin1"], init_lora_weights=False)
         peft_model = get_peft_model(base_model, config0, "adapter0").eval()
         peft_model.add_adapter("adapter1", config1)
+        inputs = {"X": torch.arange(90).view(-1, 10).to(self.torch_device)}
+        self.run_checks(peft_model, inputs)
 
+    def test_mixed_adapter_batches_lora_different_classifiers(self, mlp_lora):
```

Comment:
Depending on what is decided implementation-wise, it would be good to have additional test(s):

- If embedding is supported, let's add a test with `ModelEmbConv1D`.
- If only limited module types are supported, let's add a test that check that an error with appropriate error message is raised if the wrong type is used.

Feel free to ask questions if you need help.